### DISCUSSION TITLE
qfield take a photo problem

### DISCUSSION THREAD

**Discussion by samet2135** [opengisch/QField#5038] [Q&A]
While collecting data in the field via qfield, the photo taking tab does not appear. Does anyone know why?

**Comment by ThomasDV1984** [comment]
Has it worked in the past? Or is it a new project?

**Comment by samet2135** [comment]
I just started using the Qfield application. I created a new project and
encountered a similar problem.

16 Şub 2024 Cum 22:31 tarihinde ThomasDV1984 ***@***.***>
şunu yazdı:

**Reply by ThomasDV1984** [reply]
If it is a public project u can add me my username is voske1984, and i will have a look at it.
I haven't had a simular problem but can you see the photo widget in qgis when u add an oblect? Can you shrae a printscreen of the qgis form and the qfield form?

**Comment by samet2135** [comment]
As you can see in the attached photo, it says enter information with photo,
but unfortunately there is no tab for taking photos. It allows me to
digitize the relevant location, but the take photo tab does not appear.

17 Şub 2024 Cmt 17:35 tarihinde ThomasDV1984 ***@***.***>
şunu yazdı:

**Reply by SeqLaz** [reply]
Hello @samet2135 ,

To properly configure the project for attachments, please follow the official documentation in this link: https://docs.qfield.org/how-to/attributes-form/#configure-attachment-widget

Also, please tell us the version of QField and QGIS you are using.

Please show us the configuration of the widget in QGIS.

Please provide us screenshots of the issue you are facing.

Best Regards.

**Reply by ThomasDV1984** [reply]
Cont't see the attached photo
